src/core/initialize.cpp

```diff
@@ -88,6 +90,8 @@ void on_program_start() {
   cuda_init();
 #endif
 
+  std::signal(SIGINT, [](int) { std::abort(); });
```

Comment:
We could also call `std::exit(EXIT_FAILURE)` here, but it doesn't really matter, does it?

Reply:
We can only do `abort`. `exit` is not signal-safe.